### PR TITLE
Post-round: Get member group

### PR DESCRIPTION
**Pull request by cool-ant**

### CODE REVIEW

File: packages/webapp/src/members/api/eden-contract.ts [hunk 1]
```diff
@@ -13,10 +13,13 @@ import { devUseFixtureData } from "config";
 import { fixtureEdenMembers, fixtureMembersStats } from "./fixtures";
 
 export const getEdenMember = (account: string) => {
+    console.info(`getEdenMember().account[${account}]`);
     if (devUseFixtureData) {
         const edenMember = fixtureEdenMembers.find(
             (member) => member.account === account
         );
+        console.info(`devFixtureData; edenMember:`);
+        console.info(edenMember);
```
Comment (cool-ant) [inline packages/webapp/src/members/api/eden-contract.ts:22]: leaving these in until I add the no-consensus scenario; then I'll remove them

File: packages/webapp/src/members/api/fixtures.ts [hunk 2]
```diff
@@ -446,3 +446,19 @@ export const fixtureMembersStats: MemberStats = {
...
     ranks: [7, 4, 1, 1, 1],
 };
+
+export const fixtureMembersInGroup = (
+    round: number,
+    delegateAccount: string
+) => {
+    console.info(
+        `fixtureMembersInGroup().round[${round}], delegateAccount[${delegateAccount}]`
+    );
```
Comment (cool-ant) [inline packages/webapp/src/members/api/fixtures.ts:456]: same here

File: packages/webapp/src/elections/api/eden-contract.ts [hunk 3]
```diff
@@ -145,6 +152,51 @@ const getVoteDataRows = async (
 
 export const getVoteData = getVoteDataRows;
 
+const getCommonDelegateAccountForGroupWithThisMember = (
+    round: number,
+    member?: EdenMember
```
Comment (brandonfancher) [inline packages/webapp/src/elections/api/eden-contract.ts:157]: Looks like everything calling this function _has_ the `member`. Functions calling this function have the member as a required param. Does this need to be optional here?

**Review by brandonfancher — APPROVED** [review]

File: packages/webapp/src/elections/api/eden-contract.ts (hunk 3 above)
Comment (cool-ant) [reply packages/webapp/src/elections/api/eden-contract.ts]: good call; question mark removed

**Pull request merged**